compiler/quilt/test/test_import.py

```diff
@@ -110,6 +186,26 @@ def test_multiple_package_dirs(self):
         # Imports should find the second package
         from quilt.data.foo.nested import dataframes
 
+    def test_team_multiple_package_dirs(self):
```

Comment:
This test and the one it's copying are out-of-date. Fixing the other one might be out of scope for this PR, but there's no point in adding this test because changing the directory where build is run no longer changes the PackageStore directory.

Reply:
Nix the test, or make a better one?

Reply:
Yeah, just nix the additional test.